2024-10-13 15:01:04,034 - bot - INFO - 91 - __init__.py - localdb updated...
2024-10-13 15:01:04,075 - bot - INFO - 97 - __init__.py - 
Developed by

 ‚ñÑ‚ñÑ‚ñÑ‚ñÑ    ‚ñà‚ñà‚ñì  ‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà  ‚ñà‚ñà‚ñë ‚ñà‚ñà  ‚ñÑ‚ñÑ‚ñÑ       ‚ñà‚ñà‚ñì    
‚ñì‚ñà‚ñà‚ñà‚ñà‚ñà‚ñÑ ‚ñì‚ñà‚ñà‚ñí‚ñí‚ñà‚ñà    ‚ñí ‚ñì‚ñà‚ñà‚ñë ‚ñà‚ñà‚ñí‚ñí‚ñà‚ñà‚ñà‚ñà‚ñÑ    ‚ñì‚ñà‚ñà‚ñí    
‚ñí‚ñà‚ñà‚ñí ‚ñÑ‚ñà‚ñà‚ñí‚ñà‚ñà‚ñí‚ñë ‚ñì‚ñà‚ñà‚ñÑ   ‚ñí‚ñà‚ñà‚ñÄ‚ñÄ‚ñà‚ñà‚ñë‚ñí‚ñà‚ñà  ‚ñÄ‚ñà‚ñÑ  ‚ñí‚ñà‚ñà‚ñë    
‚ñí‚ñà‚ñà‚ñë‚ñà‚ñÄ  ‚ñë‚ñà‚ñà‚ñë  ‚ñí   ‚ñà‚ñà‚ñí‚ñë‚ñì‚ñà ‚ñë‚ñà‚ñà ‚ñë‚ñà‚ñà‚ñÑ‚ñÑ‚ñÑ‚ñÑ‚ñà‚ñà ‚ñí‚ñà‚ñà‚ñë    
‚ñë‚ñì‚ñà  ‚ñÄ‚ñà‚ñì‚ñë‚ñà‚ñà‚ñë‚ñí‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñí‚ñí‚ñë‚ñì‚ñà‚ñí‚ñë‚ñà‚ñà‚ñì ‚ñì‚ñà   ‚ñì‚ñà‚ñà‚ñí‚ñë‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñí
‚ñë‚ñí‚ñì‚ñà‚ñà‚ñà‚ñÄ‚ñí‚ñë‚ñì  ‚ñí ‚ñí‚ñì‚ñí ‚ñí ‚ñë ‚ñí ‚ñë‚ñë‚ñí‚ñë‚ñí ‚ñí‚ñí   ‚ñì‚ñí‚ñà‚ñë‚ñë ‚ñí‚ñë‚ñì  ‚ñë
‚ñí‚ñë‚ñí   ‚ñë  ‚ñí ‚ñë‚ñë ‚ñë‚ñí  ‚ñë ‚ñë ‚ñí ‚ñë‚ñí‚ñë ‚ñë  ‚ñí   ‚ñí‚ñí ‚ñë‚ñë ‚ñë ‚ñí  ‚ñë
 ‚ñë    ‚ñë  ‚ñí ‚ñë‚ñë  ‚ñë  ‚ñë   ‚ñë  ‚ñë‚ñë ‚ñë  ‚ñë   ‚ñí     ‚ñë ‚ñë   
 ‚ñë       ‚ñë        ‚ñë   ‚ñë  ‚ñë  ‚ñë      ‚ñë  ‚ñë    ‚ñë  ‚ñë
      ‚ñë                                        
                            Library python-telegram-bot

2024-10-13 15:01:04,085 - werkzeug - INFO - 97 - _internal.py - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://172.31.196.46:8080
2024-10-13 15:01:04,086 - werkzeug - INFO - 97 - _internal.py - [33mPress CTRL+C to quit[0m
2024-10-13 15:01:04,548 - bot - INFO - 22 - __init__.py - Bot commands updated!
2024-10-13 15:01:04,961 - werkzeug - INFO - 97 - _internal.py - 172.31.196.46 - - [13/Oct/2024 15:01:04] "GET / HTTP/1.1" 200 -
2024-10-13 15:01:05,163 - pyrogram.crypto.aes - INFO - 26 - aes.py - Using TgCrypto
2024-10-13 15:01:06,000 - bot - INFO - 67 - mongodb.py - Finding bot_docs in db...
2024-10-13 15:01:07,193 - bot - INFO - 74 - mongodb.py - Data found...
2024-10-13 15:01:07,194 - bot - INFO - 49 - mongodb.py - Finding bot_docs in db...
2024-10-13 15:01:07,388 - bot - INFO - 52 - mongodb.py - Data found...
2024-10-13 15:01:07,388 - bot - INFO - 59 - local_database.py - bot_docs updated in localdb...
2024-10-13 15:01:07,388 - bot - INFO - 26 - update_db.py - MongoDB database exist! Skiping update...
2024-10-13 15:01:07,389 - bot - ERROR - 164 - local_database.py - Localdb: 'bot_status'
2024-10-13 15:01:07,721 - bot - WARNING - 121 - main.py - https://q4kquizess.herokuapp.com/ is down or unreachable. ‚ùå - code - 404
2024-10-13 15:01:07,934 - telegram.ext.Application - INFO - 620 - _application.py - Application started
2024-10-13 15:04:07,854 - bot - WARNING - 121 - main.py - https://q4kquizess.herokuapp.com/ is down or unreachable. ‚ùå - code - 404
